apps/client/src/actions/get-next-locale.ts

```diff
@@ -0,0 +1,13 @@
+"use server";
+import { defaultLocale } from "../configs/configs";
+import { cookies } from "next/headers";
+
+export async function getNextLocale() {
+  const cookieStore = cookies();
+  const result = await cookieStore.get("NEXT_LOCALE")?.value;
+  if (!result) {
+    return defaultLocale;
+  }
+  const nextLocale = result;
```

Comment:
Because the result of NEXT_LOCALE maybe undefined(user never visited our page)
<img width="682" alt="image" src="https://github.com/user-attachments/assets/f1ec6374-913b-44bd-a519-74d254af6de5"> type undefined is not assignable to the locales('en' | 'zh-cn'), so if result is undefined it needs to return defaultLocale, which is 'en'

Reply:
at line 11, the result shouldn't be falsy, so what's the nextLocale for?

Reply:
The NEXT_LOCALE is set by next-intl. Once user have visited or changed language the language setting will be stored in the cookie.  In this case,  by getting the preferred language or default language from the cookie, the setting page select will render the responding language setting.

Reply:
... 

shouldn't we remove line 11 and 12, and just make it
` return result`?